src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/Renderer.cs

```diff
@@ -39,15 +41,15 @@ void Invalidate()
 				this.Log().Trace($"Render {renderCount++}");
 			}
 
-			var dpi = 1;
+			var scale = _host.ScaleOverride ?? 1f;
 
 			var resolution = _fbDev.ScreenSize;
 
 			width = (int)resolution.Width;
 			height = (int)resolution.Height;
 
-			var scaledWidth = (int)(width * dpi);
-			var scaledHeight = (int)(height * dpi);
+			var scaledWidth = (int)(width * scale);
+			var scaledHeight = (int)(height * scale);
 
 			var info = new SKImageInfo(scaledWidth, scaledHeight, SKImageInfo.PlatformColorType, SKAlphaType.Premul);
```

Comment:
Isn’t this supposed to be the physical size ? If I set the scale factor to 2, the physical size of the display didn’t change. If it’s device independent units then it should have gotten smaller instead.

Reply:
Looking over skimageinfo this is definitely physical pixels and there the number passed into it shouldn’t change. I think the original code was just written with misleading variable names and worked because the factor was 1. The only scaling factor that needs changing I think is for the canvas itself and nothing else.

Reply:
You are right... and I think the the `DisplayInformation` would need to be updated to apply the override too 🤔

Reply:
`DisplayInformation ` definitely needs to be updated properly, indeed.